pages/data-providers/[data-provider-id]/deposit-compliance.jsx

```diff
@@ -40,33 +40,50 @@ const SidebarContent = ({ context: { oai, originalId, authors, title } }) => {
  *       of Layout.Main and explicitly pass props.
  *       We should get rid out of it!
  */
-const DepositDates = ({
+const DepositCompliance = ({
   className,
   store,
   tag: Tag = 'main',
   ...restProps
 }) => (
   <Tag className={[styles.container, className].join(' ')} {...restProps}>
     <h1>Deposit compliance</h1>
+    {store.depositDates.complianceLevel && (
+      <Markdown>
+        {texts.compliance.body.render({
+          amount: (100 - store.depositDates.complianceLevel[1]).toFixed(1),
+        })}
+      </Markdown>
+    )}
 
-    <Card className={styles.complianceLevel} tag="section">
-      <h2>{texts.compliance.title}</h2>
-      {store.depositDates.complianceLevel ? (
-        <Markdown>
-          {texts.compliance.body.render({
-            amount: (100 - store.depositDates.complianceLevel).toFixed(1),
-          })}
-        </Markdown>
-      ) : (
-        <p>{texts.noData.body}</p>
-      )}
+    <Card className={styles.dataOverview} tag="section">
+      <h2>{texts.dataOverview.title}</h2>
+      <div className={styles.numbers}>
+        <NumericValue
+          tag="div"
+          value={valueOrDefault(
+            store.depositDates.complianceLevel[0],
+            'Loading...'
+          )}
+          caption="outputs counted"
+        />
+        <NumericValue
+          tag="div"
+          value={valueOrDefault('2.7', 'Loading...')}
+          append="%"
+          caption="non-compliant"
+        />
+      </div>
+      <Button tag="a" href="#deposit-dates-card" variant="contained">
+        {texts.dataOverview.redirect}
+      </Button>
     </Card>
 
     <Card className={styles.export} tag="section">
       <h2>{texts.exporting.title}</h2>
       <p>
         {texts.exporting.description.render({
-          count: store.depositDates.depositDatesCount || '',
+          count: store.depositDates.complianceLevel[0] || '',
```

Comment:
~Why it's `[0]` here. It's not really representative.~ Check the comment in `store`.